### PR TITLE
Housing spotlight

### PR DESCRIPTION
**Pull request by s-bessey**
Adds housing spotlight page

### CODE REVIEW

File: src/views/spotlights/housing/dashboard.vue [hunk 1]
```diff
@@ -1,46 +1,255 @@
...
+        :text="text"
+      />
+    </template>
+  </DashboardCard>
+  <DashboardCard>
+    <template #title
+      >What can I do to reduce the spread of COVID-19 in homes?</template
+    >
+    <template #content>
+      <div class="content">
```
Comment (eldu) [inline src/views/spotlights/housing/dashboard.vue:132]: Let's add ini some padding so the content is indented in

```suggestion
      <div class="content px-4">
```

File: src/views/spotlights/housing/dashboard.vue [hunk 2]
```diff
@@ -1,46 +1,255 @@
...
+              </tr>
+            </tbody>
+          </table>
+        </div>
+      </div>
     </template>
   </DashboardCard>
   <DashboardCard width="full">
     <template #content>
       <ControlPanel
```
Comment (eldu) [inline src/views/spotlights/housing/dashboard.vue:65]: In the ControlPanel file, can you add in the class "label" to the Label there? It gives some extra spacing.

```vue
      <label :for="type" class="label is-family-primary"
        ><strong>{{ options.label }}</strong></label
      >
```

File: src/views/spotlights/housing/dashboard.vue [hunk 3]
```diff
@@ -1,46 +1,255 @@
...
+        What is an age-adjusted rate? Age plays a big role in your risk for the
+        virus. Different groups of people have different age distributions -
+        meaning one group may have more old people, or one group may have more
+        young people. Age adjusting makes it so we can compare between groups
+        that have different age distributions.
+      </div>
+      <div v-else-if="caseControls.age.value === 'specific'">
+        Rates tell us how many people have tested positive for COVID-19 compared
+        to a larger group taking into account the population size of that group.
+      </div>
```
Comment (eldu) [inline src/views/spotlights/housing/dashboard.vue:112]: Actually!! Can this text be put into the variable dropDowns as `description` or something? That way the control panel can handle the display of this text instead. Let me know if that makes any sense!

**Review by eldu — CHANGES_REQUESTED** [review]
Okay I made a lot of comments... This is absolutely amazing work! Thanks so much Sam!

I got some weird bug? The help text isn't showing when I choose these options
Screen Shot 2023-02-17 at 10 14 11 AM

File: src/components/dashboard/BarChart.vue [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+interface Props {
+  activeStats: SpotlightStat[];
+  metric: FocusStat;
+}
+
+const props = defineProps<Props>();
+
+const spec = computed(() => {
+  return {
+    // TODO: include numbers on chart
```
[the remark this answers is not in the document]
Comment (s-bessey) [reply src/components/dashboard/BarChart.vue:19]: @maxwellkrieger Do you think we need actual numbers here? I feel like they'd be more confusing than helpful.

File: src/components/dashboard/BarChart.vue [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+    mark: "bar",
+    title: `Age-Adjusted Rate of ${props.metric.name} Per 1,000 People`,
+    encoding: {
+      y: {
+        field: "age_adjusted_rate",
+        type: "quantitative",
+        axis: {
+          labelFontSize: 12,
+          titleFontSize: 15,
+          title: "Rate Per 1,000", // TODO make "case" instead of "cases"
```
[the remark this answers is not in the document]
Comment (s-bessey) [reply src/components/dashboard/BarChart.vue:34]: Nope!

File: src/views/spotlights/housing/dashboard.vue [hunk 6]
```diff
@@ -1,46 +1,255 @@
...
+              <tr>
+                <td>
+                  Congregate
+                  <p style="font-size: 1rem">
+                    <em>*not included in this analysis</em>
+                  </p>
+                </td>
+                <td>
+                  Individuals typically not related to each other living in
+                  close proximity and/or sharing living facilities (e.g.,
```
[the remark this answers is not in the document]
Comment (s-bessey) [reply src/views/spotlights/housing/dashboard.vue:52]: Sending text as-is to RIDOH here, we'll see what they say

**Comment by eldu** [comment]
Can we keep the colors consistent for both of these graphs? Feedback from our GSDC meeting~ ❤️ 💟 😍 

Screenshot 2023-02-23 at 10 23 23 AM
Screenshot 2023-02-23 at 10 23 18 AM

**Review by eldu — APPROVED** [review]
Thanks so much Sam! This looks great~

**Pull request merged**